pytest_ethereum/testing.py

```diff
@@ -0,0 +1,67 @@
+from typing import Any, Dict
+
+from eth_tester.exceptions import TransactionFailed
+import pytest
+from web3.contract import ContractEvent
+
+from pytest_ethereum.exceptions import LogError
+
+TxReceipt = Dict[str, Any]
+
+
+class Log:
+    def __init__(
+        self, contract_event: ContractEvent, *args: Any, **kwargs: Any
+    ) -> None:
+        self.event = contract_event()
+        self.args = args
+        self.kwargs = kwargs
```

Comment:
Hmm - ok, correct me if i'm wrong, but the reason for this would be so that both of these are valid, which right now only the first one is "valid". (assuming they match the emitted log) 
```python
Log(contract_event, b'1').is_present(receipt) 
Log(contract_event, first=b'1').is_present(receipt)
```
where would you suggest modifying `merge_args_and_kwargs` to allow for missing values, in a copy of the function in `pytest_ethereum/utils/`?

Reply:
> in a copy of the function in `pytest_ethereum/utils/`

Yes (and it should be moved to `pytest_ethereum/_utils` (with the `_` prefix)